### PR TITLE
Личный проект: начинаем программировать

### PR DESCRIPTION
**Pull request by katasonova**
---
:mortar_board: [Личный проект: начинаем программировать](https://up.htmlacademy.ru/react/3/user/579401/tasks/5)

### CODE REVIEW

File: src/components/app/app.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Main from '../main/main.jsx';
+
+const App = (props) => {
+  // eslint-disable-next-line react/prop-types
+  return <Main MovieTitle={props.MovieTitle} MovieGenre={props.MovieGenre} MovieReleaseYears={props.MovieReleaseYears}/>;
```
Comment (artrudov) [inline src/components/app/app.jsx]: A property name is written in camelCase

File: src/components/main/main.jsx [hunk 2]
```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+
+const Main = (props) => {
+  return (<React.Fragment>
```
Comment (artrudov) [inline src/components/main/main.jsx]: If you use `React.Fragment` you need import without React, for example `import React, {Fragment} from 'react';`

You should use Fragment when you need to use the key attribute. You usually have to use `<></>`. For example `<>You JSX code</>`

File: src/components/main/main.jsx [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+            <p className="movie-card__meta">
+              <span className="movie-card__genre">{
+                // eslint-disable-next-line react/prop-types
+                props.MovieGenre}</span>
+              <span className="movie-card__year">{
+                // eslint-disable-next-line react/prop-types
+                props.MovieReleaseYears}</span>
+            </p>
+
+            {/* <div className="movie-card__buttons">
```
Comment (artrudov) [inline src/components/main/main.jsx]: Why is it commented?

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/app/app.jsx';
+
+ReactDOM.render(<App
+  MovieTitle={`Harry Potter`} MovieGenre={`Fantasy`} MovieReleaseYears={2020}
```
Comment (artrudov) [inline src/index.js]: Harry Potter + Fantasy + 2020 are constants

File: src/components/main/main.jsx (hunk 3 above)
Comment (katasonova) [reply src/components/main/main.jsx]: it caused some linting issues, so i commented it out for a while

**Pull request merged**